news/_posts/2016-10-15-kernel-release-cycle-16.md

```diff
@@ -0,0 +1,33 @@
+---
+author: "@dlech"
+title: "Kernel Release Cycle 16"
+---
+
+Nothing too exciting in this release. Most of the changes are for adding support
+for a new BeagleBone cape that is being developed.
+
+Also support for polling the `status` attribute of the tacho motor class has
```

Comment:
Could you change this opening sentence of this paragraph to make it flow better? Something like "This version adds support for the..."